### PR TITLE
feat: added dnd and delete option in kanban

### PR DESCRIPTION
**Pull request by SreehariJayaraj**

### CODE REVIEW

File: pnpm-lock.yaml
Comment (DarkPhoenix2704) [inline pnpm-lock.yaml:1]: Revert this. You are downgrading the pnpm-lock file

**Review by DarkPhoenix2704 — CHANGES_REQUESTED** [review]
Instead of cascade, this can be a series of manual operation. Using cascade can cause unexpected data loss in future

Also, if knex migration is still in plan, we can avoid this

File: pnpm-lock.yaml
Comment (sreehari2003) [reply pnpm-lock.yaml:1]: ok, can you do the Knex implementation? , we can incrementally migrate to knex.

File: apps/web/views/tasks/Kanban.tsx [hunk 1]
```diff
@@ -47,11 +62,43 @@ export const Kanban = ({ title, _count, id, tasks }: KanbanType) => {
...
+    const [{ isOver }, drop] = useDrop(() => ({
+        accept: 'TASK',
+        collect: (monitor) => ({
+            isOver: monitor.isOver(),
+            canDrop: monitor.canDrop(),
+        }),
+        drop: (item: any, monitor) => {
+            if (monitor.didDrop()) return;
+
+            if (item.id === id) return;
```
Comment (sreehari2003) [inline apps/web/views/tasks/Kanban.tsx:74]: logic is wrong

**Comment by DarkPhoenix2704** [comment]
THis still needs to be merged?  @sreehari2003

**Comment by SreehariJayaraj** [comment]
> THis still needs to be merged? @sreehari2003

no need

**Pull request closed without merging**